.github/workflows/automated-tests.yml

```diff
@@ -56,3 +56,17 @@ jobs:
           go-version: "1.20"
       - name: E2E tests
         run: make test-e2e-short-cometmock
+  Trace-Tests:
```

Comment:
Add a test that reads traces to the CI to check that trace-reading does not break. Could be expanded in the future